apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js

```diff
@@ -110,88 +104,57 @@ class RequestAssignment extends React.Component {
             input={
               <DateInput
                 name="eta"
-                value={this.state.eta}
-                onChange={this.changeDate}
+                value={eta}
+                onChange={updateEta}
+                width="100%"
```

Comment:
For `HoursInput` we use `wide`, for `DateInput` we use `width="100%"`. It might be out-of-scope to change this, but reading through this code right now makes me wonder if they're functionally equivalent.

Reply:
They're functionally equivalent, but DateInput lives in our repo and was.. adjusted. It needs to be updated - but that's for the cooling period/refactoring overhaul.